graylog2-web-interface/src/components/event-definitions/event-definition-form/NotificationSettingsForm.jsx

```diff
@@ -41,9 +43,17 @@ class NotificationSettingsForm extends React.Component {
     this.setState(stateUpdate);
   };
 
+  handleBacklogSizeChange = (e) => {
+    const { eventDefinition, onSettingsChange } = this.props;
+    const nextNotificationSettings = lodash.cloneDeep(eventDefinition.notification_settings);
+    nextNotificationSettings.backlog_size = e.target.value;
```

Comment:
HTML `<input>`s always keep values as strings, and this is not converting the input into a number, which is what the API expects, so the request may fail or end up storing something wrong in the DB. Please use `FormsUtils.getValueFromInput()` to get a converted value from the input.